panda/src/audio/audioSound.h

```diff
@@ -19,6 +19,7 @@
 #include "typedReferenceCount.h"
 #include "pointerTo.h"
 #include "filterProperties.h"
+#include "nodePath.h"
```

Comment:
Included but not used

Reply:
I added nodePath.h so that I can use LVector3. Is there a better way?

Reply:
Yes, include `lvector3.h`, or `luse.h` to pull in all linmath classes. :)